src/arkode/arkode_lsrkstep.c

```diff
@@ -423,9 +419,25 @@ int lsrkStep_FullRHS(ARKodeMem ark_mem, sunrealtype t, N_Vector y, N_Vector f,
     break;
 
   case ARK_FULLRHS_END:
-    /* No further action is needed since the currently
-	    available methods evaluate the RHS at the end of each time step*/
+    /* No further action is needed if STS since the currently
+	    available STS methods evaluate the RHS at the end of each time step
```

Comment:
Not sure why the format check didn't catch this, replace tab with spaces
```suggestion
    /* No further action is needed if STS since the currently
       available STS methods evaluate the RHS at the end of each time step
```

Reply:
It is indeed space on my local machine. I will make it even worse to make sure formatting will catch and fix it.